test/model/test_node.py

```diff
@@ -766,3 +766,32 @@ async def test_firmware_events(wallmote_central_scene: Node):
         == FirmwareUpdateStatus.OK_RESTART_PENDING
     )
     assert event.data["firmware_update_finished"].wait_time == 10
+
+
+async def test_value_added(climate_radio_thermostat_ct100_plus):
+    """Test value added event."""
+    node: Node = climate_radio_thermostat_ct100_plus
+    event = Event(
+        "value added",
+        {
+            "source": "node",
+            "event": "value added",
+            "nodeId": node.node_id,
+            "args": {
+                "commandClassName": "Battery",
+                "commandClass": 128,
+                "endpoint": 1,
+                "property": "isHigh",
+                "propertyName": "isHigh",
+                "metadata": {
+                    "type": "boolean",
+                    "readable": True,
+                    "writeable": False,
+                    "label": "Low battery level",
+                },
+                "value": True,
+            },
+        },
+    )
+    node.receive_event(event)
+    assert f"{node.node_id}-128-1-isHigh" in node.values
```

Comment:
I don't think this fixture sets up the test case properly. Looks like the Low Battery level already has a value in the fixture data. What you really want is a Node value with a missing `value`, and then assert that the Value instance is the same before and after the event. Do you need new test data for that, or could you just delete the value from the fixture data?

I could provide a dump with the missing value, and then the "value added" message if needed.

Reply:
I faked it.